gui/src/components/Taipy/Chart.tsx

```diff
@@ -118,6 +118,38 @@ const getValueFromCol = (values: TraceValueType | undefined, col: string): (stri
     return [];
 };
 
+const getDecimatorPayload = (
+    decimator: string | undefined,
+    plotRef: React.RefObject<HTMLDivElement>,
+    modes: string[],
+    columns: Record<string, ColumnDesc>,
+    traces: string[][],
+    relayoutData: PlotRelayoutEvent | undefined = undefined
+) => {
+    return decimator
+        ? {
+              width: plotRef.current?.clientWidth,
+              height: plotRef.current?.clientHeight,
+              xAxis: traces.length && traces[0].length && traces[0][0] && columns[traces[0][0]].dfid,
```

Comment:
I would still make a function of this (3 calls)
And this way the value can be set to `undefined` instead of `false` when there's no data for the axis